lib/core/FileHandlerAsyncTask.js

```diff
@@ -9,7 +9,7 @@ var FileHandleGenerator = function(fileName, truncate) {
 var _execute = function(callback) {
 	this.callback = callback;
 	var self = this;
-	var mode = 'a+';
+	var mode = 'r+';
```

Comment:
Quote from: https://nodejs.org/api/fs.html#fs_fs_open_path_flags_mode_callback
>'r+' - Open file for reading and writing. An exception occurs if the file does not exist.
For new downloads, I will have to create a file first and then open it in r+ mode.